cmd/account/transfer.go

```diff
@@ -17,15 +17,23 @@ import (
 )
 
 var transferCmd = &cobra.Command{
-	Use:     "transfer <amount> <to>",
+	Use:     "transfer <amount> [<denom>] <to>",
```

Comment:
Interesting usage. I was thinking of introducing a slightly different flow for setting the validator's commission fee without the space character (e.g. `--bounds 329000/2%/4%`). A separate denominator argument makes sense here though. :+1: